rest/model/build.go

```diff
@@ -112,6 +112,8 @@ func (apiBuild *APIBuild) BuildFromService(v build.Build, pp *model.ParserProjec
 		origin = triggerAdHoc
 	case evergreen.GitTagRequester:
 		origin = gitTagOrigin
+	case evergreen.GithubMergeRequester:
+		origin = patchOrigin
```

Comment:
Can we add a unit test?

Reply:
I'm actually going to leave this out for now, and revisit in a later ticket, since I don't have a ton of context of why we're doing this.